CONTRIBUTING.md

```diff
@@ -1,10 +1,113 @@
-#Contributing
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+[Code of Conduct](#code-of-conduct) 
 
-For contributing please follow the instructions below:
-1. Fork the repository on GitHub
-2. Clone the project to your own machine
-3. Commit changes to your own branch
-4. push your work back up to your forked repo
-5. Submit a Pull Request so that we can review your changes
+# Pull Request Process
+1. Create Issue with relevant information about your PR
+2. Fork the repository on GitHub
```

Comment:
I would add what perquisites are needed in order to start working on the project. 
An outsider wouldn't know vagrant or VirtualBox are needed.

Reply:
@ronyyosef was it resolved?